src/main/java/org/generationcp/middleware/pojos/Germplasm.java

```diff
@@ -334,13 +349,6 @@
 	@Transient
 	private String preferredAbbreviation = null;
 
-	/**
-	 * This variable is populated only when the Germplasm POJO is retrieved by using GermplasmDataManager.getGermplasmWithMethodType().
-	 * Otherwise it is null always.
-	 */
-	@Transient
-	private Method method = null;
```

Comment:
Given we've joined to Method table, we can remove `GermplasmDataManager#getGermplasmWithMethodType`